src/containers/Application.js

```diff
@@ -225,6 +232,46 @@ it is accessible by the sidebar toggler button.
 	}
 
 
+	interceptorRequest(axios, interceptor) {
+		// Add a request interceptor
+		axios.interceptors.request.use(
+			interceptor,
+			function(error) {
+				return Promise.reject(error)
+			});
+	}
+
+
+	interceptorResponse(axios, interceptor) {
```

Comment:
This is unnecessary.
It is a normal error handling, not specific to interceptors.

401 is "Not authorized" ... Token expiration is just one specific reason when this could happen.